source/replication.txt

```diff
@@ -138,8 +139,9 @@ replica set elections.
 
 When the former primary recovers from a failover event and reconnects to 
 the cluster, MongoDB may rollback any write operations that were *not*
-replicated to secondary members at the time of failure. MongoDB attempts
-to avoid rollbacks, which should be rare. 
+replicated to secondary members at the time of failure. Write operations
+issued with a :ref:`write concern <write-concern>` of 
+:writeconcern:`majority` that return successfully do not roll back.
 See :ref:`replica-set-rollbacks` for more information.
```

Comment:
I suggest striking this paragraph entirely. It's difficult to say anything meaningful here without either sending the message that rollbacks are common or that all writes should use w: majority. Addressing either issue is too in the weeds for this overview page. Instead, I suggest a simple statement pointing readers to further information about writes in replica sets. Something like, "To learn more about how writes are handled in replica sets during failover scenarios see:" and then point the reader to:
- reference/write-concern.txt
- core/replica-set-rollbacks.txt

Reply:
Done